wearApp/build.gradle.kts

```diff
@@ -13,17 +13,17 @@ android {
     defaultConfig {
         applicationId = "dev.johnoreilly.gemini"
         minSdk = 30
-        targetSdk = 33
+        targetSdk = 34
```

Comment:
targetSdk should be 33. 

https://developer.android.com/google/play/requirements/target-sdk

Reply:
corrected, thanks. https://github.com/joreilly/GeminiKMP/pull/16